runtime/arctic/src/lib.rs

```diff
@@ -491,30 +492,34 @@ impl pallet_ethereum::Config for Runtime {
 	type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
 }
 
-frame_support::parameter_types! {
-    pub const AssetDeposit: Balance = 500 ;
-	pub const AssetAccountDeposit: Balance = 500 ;
-	pub const MetadataDepositBase: Balance = 500 ;
-    pub const MetaDataDepositPerByte: Balance = 500 ;
-    pub const ApprovalDeposit: Balance = 500 ;
-	pub const StringLimit: u32 = 50;
-}
+pub type AssetId = u128;
+
 
+parameter_types! {
+    pub const AssetDeposit: Balance = 1_000_000;
+    pub const ApprovalDeposit: Balance = 1_000_000;
+    pub const AssetsStringLimit: u32 = 50;
+    /// Key = 32 bytes, Value = 36 bytes (32+1+1+1+1)
+    // https://github.com/paritytech/substrate/blob/069917b/frame/assets/src/lib.rs#L257L271
+    pub const MetadataDepositBase: Balance = currency::deposit(1, 68);
+    pub const MetadataDepositPerByte: Balance = currency::deposit(0, 1);
+    pub const AssetAccountDeposit: Balance = currency::deposit(1, 18);
+}
 impl pallet_assets::Config for Runtime {
-	type Event = Event;
-	type Balance = Balance;
-	type AssetId = u32;
-	type Currency = Balances;
-	type ForceOrigin = frame_system::EnsureRoot<AccountId>;
-	type AssetDeposit = AssetDeposit;
-	type MetadataDepositBase = MetadataDepositBase;
-	type MetadataDepositPerByte = MetaDataDepositPerByte;
-	type ApprovalDeposit = ApprovalDeposit;
-	type StringLimit = StringLimit;
-	type Freezer = ();
-	type Extra = ();
-	type WeightInfo = pallet_assets::weights::SubstrateWeight<Runtime>;
-	type AssetAccountDeposit = ();
+    type Event = Event;
+    type Balance = Balance;
+    type AssetId = AssetId;
```

Comment:
Are we sure about the change from u32 to u128?

Reply:
I am not sure why we added u32, but it is a recommended assetId type in vanilla template, as there can be long assets id also in the mainnet, so I think its fine to have like this